src/index.js

```diff
@@ -7,6 +7,7 @@ import {BrowserRouter} from "react-router-dom";
 import {NavBar} from "./components/NavBar/NavBar";
 import {DataProvider,useData} from "./contexts/product-context";
 import {Footer} from "./components/Footer/Footer";
+import {CartProvider,useCart} from "./contexts/cart-context"
```

Comment:
Have separate index.js files for different things, eg: components folder will have one index.js, contexts folder will have separate index.js file, don't import all in main index.js file which is at root level. [Read more about it](https://medium.com/@klauskpm/do-a-barrel-export-aa5b79b76b05).